tests/sentry/deletions/test_group.py

```diff
@@ -305,30 +306,42 @@ def select_rows(
     def tenant_ids(self) -> dict[str, str]:
         return {"referrer": self.referrer, "organization_id": self.organization.id}
 
-    def test_issue_platform(self) -> None:
+    def test_simple_issue_platform(self) -> None:
         # Adding this query here to make sure that the cache is not being used
         assert self.select_error_events(self.project.id) is None
         assert self.select_issue_platform_events(self.project.id) is None
+
         # Create initial error event and occurrence related to it; two different groups will exist
         event = self.store_event(data={}, project_id=self.project.id)
-        occurrence, group_info = self.create_occurrence(event, type_id=FeedbackGroup.type_id)
+        # XXX: We need a different way of creating occurrences which will insert into the nodestore
```

Comment:
I have noticed that the code path for `create_occurrence` does not insert anything in the nodestore. I will have to investigate this at a later time.